app/routes.py

```diff
@@ -34,34 +36,48 @@ def checker_page(ver):
     return make_response(jsonify({'success': True, 'message': 'OK', 'version': APP_VERSION}))
 
 
-@app.route('/<int:r>,<int:g>,<int:b>/<string:filename>', methods=['GET'])
-def color(ver, r, g, b, filename):  # pylint: disable=invalid-name
+@app.route('/all', methods=['GET'])
+def all_images(ver):
+    """
+    List all available icons and return them as a JSON array
+    When available icons can be colorized, it will output URLs with the default color red
+    """
+    check_version(ver)
+    return make_response(jsonify(get_all_icons(request.base_url.replace('/all', ''))))
```

Comment:
I've added that, I just add to do some substring operation, otherwise there was two slashes side by side in the URL (and then routes weren't recognized)